Umbraco-Cloud/Upgrades/Upgrading-from-9-to-10/index.md

```diff
@@ -129,29 +129,29 @@ With the packages and projects updated, it is time to make some changes to some
     "$schema": "./appsettings-schema.json",
     ```
 
-Apply this change to the following files as well:
-  - **appsettings.Development.json**
-  - **appsettings.Production.json**
-  - **appsettings.Staging.json**
+    Apply this change to the following files as well:
+    * **appsettings.Development.json**
+    * **appsettings.Production.json**
+    * **appsettings.Staging.json**
 
 3. The final thing you need to do before testing the upgrade locally, is to remove a series of files no longer needed in your project.
 
-Remove the following files and folders *manually* from your local project:
+    Remove the following files and folders *manually* from your local project:
 
-* `/wwwroot/umbraco`
-* `/umbraco/PartialViewMacros`
-* `/umbraco/UmbracoBackOffice`
-* `/umbraco/UmbracoInstall`
-* `/umbraco/UmbracoWebsite`
-* `/umbraco/config/lang`
+    * `/wwwroot/umbraco`
+    * `/umbraco/PartialViewMacros`
+    * `/umbraco/UmbracoBackOffice`
+    * `/umbraco/UmbracoInstall`
+    * `/umbraco/UmbracoWebsite`
+    * `/umbraco/config/lang`
 
-If using Umbraco Forms, update your files and folders according to the [Upgrading - version specific](../../../Add-ons/UmbracoForms/Installation/Version-Specific.md) for version 10 article.
+4. If using Umbraco Forms, update your files and folders according to the [Upgrading - version specific](../../../Add-ons/UmbracoForms/Installation/Version-Specific.md) for version 10 article.
 
-By default, Umbraco Deploy will create an SQLite database. If you want to re-use the existing LocalDB database, configure the [ConnectionStrings](https://our.umbraco.com/documentation/Add-ons/Umbraco-Deploy/Upgrades/version-specific#database-initialization) or use the [`PreferLocalDbConnectionString` setting](https://our.umbraco.com/documentation/Add-ons/Umbraco-Deploy/Deploy-Settings/#preferlocaldbconnectionstring).
+5. By default, Umbraco Deploy will create an SQLite database. If you want to re-use the existing LocalDB database, configure the [ConnectionStrings](https://our.umbraco.com/documentation/Add-ons/Umbraco-Deploy/Upgrades/version-specific#database-initialization) or use the [`PreferLocalDbConnectionString` setting](https://our.umbraco.com/documentation/Add-ons/Umbraco-Deploy/Deploy-Settings/#preferlocaldbconnectionstring).
```

Comment:
This is not really an action 🤔 
Could this be split up in a way?
Like:

5. Choose your Database configuration:
    * To re-use the existing LocalDB database configure the [ConnectionStrings](https://our.umbraco.com/documentation/Add-ons/Umbraco-Deploy/Upgrades/version-specific#database-initialization) or use the [`PreferLocalDbConnectionString` setting](https://our.umbraco.com/documentation/Add-ons/Umbraco-Deploy/Deploy-Settings/#preferlocaldbconnectionstring)
    * To use the default configuration skip this step.

Not sure what would work best 🤔 but I don't think there should be two different options in the same step.

Reply:
I like the idea of splitting the step and providing specific actions. Will change it as above,